src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -172,4 +178,59 @@ private function getAvailableFormatOptions(): array
     {
         return (new DescriptorHelper())->getFormats();
     }
+
+    private function sortBy(RouteCollection $routeCollection, string $column): RouteCollection
+    {
+        $routesArray = $routeCollection->all();
+        $sortedRoutes = new RouteCollection();
+
+        if ('name' === $column) {
+            \ksort($routesArray);
+            foreach ($routesArray as $name => $route) {
+                $sortedRoutes->add($name, $route);
+                $sortedRoutes->addAlias($route->getDefault('_controller'), $name);
```

Comment:
I don't get why you add an alias to the controller for every routes.

Not all routes have a controller. Example: `_logout_main`.
> Symfony\Component\Routing\RouteCollection::addAlias(): Argument #1 ($name) must be of type string, null given, called in src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php on line 192